File: 04-mad-lib/MadLib.java
class MadLib {
  public static void main(String[] args) {
    var noun = System.console().readLine("Enter a noun: ");
    var verb = System.console().readLine("Enter a verb: ");
    var adjective = System.console().readLine("Enter an adjective: ");
    var adverb = System.console().readLine("Enter an adverb: ");

    System.out.printf("Do you %s your %s %s %s? That's hilarious!%n", verb, adjective, noun, adverb);
  }
}

File: 04-mad-lib/mad-lib.js
const readline = require('readline');
const rl = readline.createInterface(process.stdin, process.stdout);

rl.question('Enter a noun: ', (noun) => {
  rl.question('Enter a verb: ', (verb) => {
    rl.question('Enter an adjective: ', (adjective) => {
      rl.question('Enter an adverb: ', (adverb) => {
        console.log(`Do you ${verb} ${adjective} ${noun} ${adverb}? That's hilarious!`);
        rl.close();
      });
    });
  });
});
